app/models/linkables.rb

```diff
@@ -16,14 +16,21 @@ def organisations
     @organisations ||= for_document_type('organisation')
   end
 
+  def needs
+    @needs ||= for_document_type('need', include_draft: false)
```

Comment:
When they were included, there were a lot more as there are currently lots of draft/proposed needs in Maslow/Need API. So excluding draft content reduces the number to select from.

Also, if it were working, I believe the info frontend only shows validated (published) needs when they are associated with content, and this will be true for the Publishing API as well, as I believe link expansion will ignore draft content (in this case the need) when presenting to the live content store.

I'm not fussed either way on this, as its more of a product decision in an area I still don't have a lot of knowledge about.